scalyr_agent/util.py

```diff
@@ -1218,7 +1218,7 @@ def is_alive(self):
 
 class RateLimiter(object):
     """An abstraction that can be used to enforce some sort of rate limit, expressed as a maximum number
-    of bytes to be consumed over a period of time.
+    of bytes to be consumed over a period of time. This abstraction is not thread-safe.
```

Comment:
Put the comment about it not being thread-safe on a separate line to draw more attention to it.